### PR TITLE
812 deploy contract on e2e test start

### PR DESCRIPTION
**Pull request by ognjenkurtic**
# Description

Adds a deploy script for the BPI contract. 
Adds a bash script to run hardhat node and deploy contract prior to executing the e2e test. 
Adds hardhat node start and deploy contracts to GH action.

## Related Issue

#812 

## Motivation and Context

e2e tests must cover the CCSM part as well.

## How Has This Been Tested

Ran the bash script locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Comment by ognjenkurtic** [comment]
Not ready for review yet, need to verify GH action works and update docs.

**Comment by ognjenkurtic** [comment]
Ready for review @biscuitdey @Therecanbeonlyone1969

File: .github/workflows/bri-3.yml [hunk 1]
```diff
@@ -108,6 +108,16 @@ jobs:
       - name: Run prisma db seed
         working-directory: examples/bri-3
         run: npx prisma db seed
+      
+      - name: Install hardhar dependencies
```
Comment (Therecanbeonlyone1969) [inline .github/workflows/bri-3.yml:112]:
```suggestion
      - name: Install hardhat dependencies
```

**Review by Therecanbeonlyone1969 — APPROVED** [review]
LGTM

**Pull request merged**